ChiModules/KEigenvalueSolver/initialize_k_solver.cc

```diff
@@ -11,7 +11,7 @@ using namespace LinearBoltzmann;
 //###################################################################
 void KEigenvalue::Solver::InitializeKSolver()
 {
-  Initialize(); // LinearBoltzmann::Solver initialize
+  LinearBoltzmann::Solver::Initialize(); // LinearBoltzmann::Solver initialize
```

Comment:
```suggestion
  LinearBoltzmann::Solver::Initialize();
```

Useless comment with the change.

Reply:
Done